bootstrap/digitalocean_k8s_centos_7_master.sh

```diff
@@ -37,8 +37,8 @@ systemctl enable docker
 systemctl enable kubelet.service
 systemctl start docker
 
-PUBLICIP=$(curl ifconfig.me)
-PRIVATEIP=$(ip addr show dev tun0 | awk '/inet / {print $2}' | cut -d"/" -f1)
+PUBLICIP=$(curl -s http://169.254.169.254/metadata/v1/interfaces/public/0/ipv4/address)
+PRIVATEIP=$(curl -s http://169.254.169.254/metadata/v1/interfaces/private/0/ipv4/address)
```

Comment:
`PRIVATEIP` is actually VPN's interface IP, not Droplet's Private IP, so we can't use metadata here. We'll have to resort back to `ip`.

Reply:
Maybe we can also rename this to something more obvious.

Reply:
How about renaming it to VPNIP?

Reply:
Great — it will do the job.